### PR TITLE
api/index.py: address warning: "alembic_version" does not ...

### PR DESCRIPTION
**Pull request by jgehrcke**
When working locally, this warning appears periodically in the containerized stack log output. Sometimes the log is central for being productive. And then this noise wastes time. 

Pragmatic solution here. A better solution with less divergence between testing and prod would be to have this table in local deployment as well as rightfully noted by @austin3dickey.

### CODE REVIEW

File: conbench/api/index.py [hunk 1]
```diff
@@ -85,20 +85,28 @@ def get(self):
...
+
+        if Config.TESTING:
+            alembic_version = "not-set-in-testing"
+        else:
+
+            query = text("SELECT version_num FROM alembic_version")
+            try:
+                # In local deployment the PostgreSQL server logs: `ERROR:  relation
+                # "alembic_version" does not exist at character 25`. That is
+                # probably because not schema migration has yet been performed?
```
Comment (jonkeane) [inline conbench/api/index.py:97]: Typo?

```suggestion
                # probably because no schema migration has yet been performed?
```

Comment (jonkeane) [reply conbench/api/index.py:97]: Also I know this isn't your text — just moving, but it stood out anyawy!

**Review by jonkeane — APPROVED** [review]
I'm good with this + thankful for the detailed comments we have + adding to them. This _could_ be confusing at some point in the future, but I think this is a good balance between making ourselves more productive + documenting the slight side step we're taking in case it bumps into us in the future.

**Review by austin3dickey — APPROVED** [review]
Nice, I like the distinction between `not-set-in-testing` and `err`.

File: conbench/api/index.py (hunk 1 above)
Comment (jgehrcke) [reply conbench/api/index.py:97]: Yes! Will change that in another branch.

**Pull request merged**